dts/bindings/input/microchip,cap12xx.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2022 Keiya Nobuta
+# SPDX-License-Identifier: Apache-2.0
+
+description: CAP12xx 3, 6 or 8-channel capacitive touch sensor
+
+compatible: "microchip,cap12xx"
+
+include: i2c-device.yaml
+
+properties:
+  int-gpios:
+    type: phandle-array
+
+  channels:
+    type: int
+    required: true
+    description: |
+      Number of channles available on the chip (3, 6 or 8)
+
+  input-codes:
+    type: array
+    required: true
+    description: |
+      Array of input event key codes (INPUT_KEY_* or INPUT_BTN_*).
+
+  repeat:
+    type: boolean
```

Comment:
would it make sense to have a default value here?

Reply:
Thanks for the review.
This is also related to your next comment, the default is set in the C file and defaults to false.
What is the preferred method?
My reasoning was: you don't specify "repeat" - it will not repeat. You add "repeat" - it repeats

Reply:
sorry I was meaning to delete this one - booleans default to false implicitly, my comment was wrong :-)